src/helm/benchmark/scenarios/entity_matching_scenario_fixed_random_state.py

```diff
@@ -0,0 +1,61 @@
+from typing import Dict, Tuple
+
+import numpy as np
+
+
+RandomState = Tuple[str, np.ndarray, int, int, float]
+
+
+# Saved numpy random states by dataset.
+_DATASET_TO_RANDOM_STATE: Dict[str, RandomState] = {
+    "Beer": (
+        "MT19937",
+        # fmt: off
+        np.array(
```

Comment:
I noticed we were using json files for some of these fixes, which seems better - should we do that here too?

Reply:
OK, I made this a JSON file too.